templates/model.txt

```diff
@@ -0,0 +1,18 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm'
+
+export default class ${filename} extends BaseModel {
+  @column({ isPrimary: true })
+  public id: number
+
+  @column.dateTime({ autoCreate: true })
+  public createdAt: DateTime
+
+  @column.dateTime({ autoCreate: true, autoUpdate: true })
+  public updatedAt: DateTime
+
+  public static $columns: Pick<
```

Comment:
Can we remove this for now. I realized this is causing too much confusion. We will add it back once we are able to smartly generate the models